src/channels/views/ChannelDetails/ChannelDetails.tsx

```diff
@@ -46,6 +50,13 @@ export const ChannelDetails: React.FC<ChannelDetailsProps> = ({ id }) => {
     onCompleted: onSubmit
   });
 
+  const [activateChannel, activateChannelOpts] = useChannelActivateMutation({});
+
+  const [
+    deactivateChannel,
+    deactivateChannelOpts
+  ] = useChannelDeactivateMutation({});
```

Comment:
What about channel activation errors? Don't we want to handle them somehow?

Reply:
I added notifications